demos/hypo/demo.js

```diff
@@ -30,9 +30,9 @@ async function displaySecurityDetails (postmanJSON) {
                 capitalGainTaxRate: 0,
                 stateIncomeTaxRate: 0,
                 dividendTaxRate: 0,
-                illustrationTrailingTimePeriod: 'EarliestCommon',
-                startDate: '1998-12-31T00:00:00.000Z',
-                endDate: '2008-12-31T00:00:00.000Z',
+                illustrationTrailingTimePeriod: 'Customized',
+                startDate: '2002-01-01',
+                endDate: '2004-01-01',
```

Comment:
Q: Why last points are always `0`? Can we do something about it? Tested different dates, and still the same problem. It looks strange on the line chart..

<img width="1049" height="382" alt="Image" src="https://github.com/user-attachments/assets/f6902c7c-b4ea-4f63-adc8-92d664442520" />

Reply:
Requested explanation from Morningstar, awaiting response..